### PR TITLE
A11y shrine: purpose of each link

### LINKED ISSUE #366: A11y shrine: purpose of each link
### Description

The purpose of each link should be clear from the link text
### Expected outcome

Every button or link text should have a clear title, name or description to describe its purpose.
### Actual outcome

There are buttons and links over the application without any text inside it. Some of them grouped elements within them or show only an icon.

### PR DESCRIPTION
**Pull request by felixzapata**
Fix #366.

### CODE REVIEW

File: templates/shrine/src/shrine-app.html [hunk 1]
```diff
@@ -223,7 +223,7 @@
           <app-toolbar class="profileBar">
             <img class="profilePic" src="//app-layout-assets.appspot.com/assets/shrine/profile_pic.jpg"  width="30" height="30">
             <div class="profileName">Stella</div>
-            <paper-icon-button icon="settings"></paper-icon-button>
+            <paper-icon-button icon="settings" alt="show settings"></paper-icon-button>
```
Comment (blasten) [inline templates/shrine/src/shrine-app.html]: Do you mind changing this to just `Settings`?

File: templates/shrine/src/shrine-app.html [hunk 2]
```diff
@@ -238,15 +238,15 @@
...
+              <paper-icon-button icon="menu" drawer-toggle alt="Toogle navigation menu"></paper-icon-button>
+              <a href="#[[sectionData.section]]" tabindex="-1"><paper-icon-button icon="arrow-back"  alt="Back to the home"></paper-icon-button></a>
             </div>
             <div class="spacer">
               <div class="logo"></div>
             </div>
             <div class="navItem">
-              <paper-icon-button icon="shopping-cart"></paper-icon-button>
-              <paper-icon-button icon="more-vert"></paper-icon-button>
+              <paper-icon-button icon="shopping-cart" alt="Show shopping cart"></paper-icon-button>
```
Comment (blasten) [inline templates/shrine/src/shrine-app.html]: Shopping cart

File: templates/shrine/src/shrine-app.html [hunk 3]
```diff
@@ -238,15 +238,15 @@
...
+              <paper-icon-button icon="more-vert" alt="Show more options"></paper-icon-button>
```
Comment (blasten) [inline templates/shrine/src/shrine-app.html]: More options

File: templates/shrine/src/shrine-detail.html [hunk 4]
```diff
@@ -235,7 +235,7 @@
         <div class="image">
           <img id="img">
           <div class="info">
-            <paper-icon-button icon="info-outline"></paper-icon-button>
+            <paper-icon-button icon="info-outline" alt="show more information about the item"></paper-icon-button>
```
Comment (blasten) [inline templates/shrine/src/shrine-detail.html]: More information about the item

File: templates/shrine/src/shrine-list.html [hunk 5]
```diff
@@ -87,7 +87,7 @@
 
     <div class="grid">
       <template is="dom-repeat" items="[[items]]" sort="_sortItems" initial-count="1">
-        <a href$="#[[section]]/[[item.id]]" class="item">
+        <a href$="#[[section]]/[[item.id]]" class="item" aria-label$="view more information about [[item.title]]">
```
Comment (blasten) [inline templates/shrine/src/shrine-list.html]: More information ...

File: templates/shrine/src/shrine-app.html (hunk 1 above)
Comment (blasten) [inline templates/shrine/src/shrine-app.html]: Also `alt` just sets `aria-label`. Can we just set `aria-label` here?

Comment (felixzapata) [inline templates/shrine/src/shrine-app.html]: @blasten I´ve used the `alt` attribute in this case according with the element documentation.

**Comment by blasten** [comment]
LGTM

**Pull request merged**